templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -34,9 +34,8 @@
     "from alpineer import io_utils, load_utils, misc_utils\n",
```

Comment:
<p>I think this would be better if condensed with the cell below. </p><p> </p><p>The first cell should contain the instructions about what a neighborhood matrix is, what the parameters are, etc, and have the user set those values. </p><p> </p><p>Then the second cell should check if the path exists; if it does, load the matrix. Otherwise, create the matrix. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>